Python/PRP/pyCreatable.cpp

```diff
@@ -104,6 +105,33 @@ PY_METHOD_VA(Creatable, write,
     Py_RETURN_NONE;
 }
 
+PY_GETSET_GETTER_DECL(Creatable, prc)
+{
+    hsRAMStream* S = new hsRAMStream();
+    pfPrcHelper* prc = new pfPrcHelper(S);
+    self->fThis->prcWrite(prc);
+
+    S->rewind();
+    size_t dataSize = S->size() - S->pos();
+    unsigned char* buf = new unsigned char[dataSize];
+    S->read(dataSize, buf);
+
+    auto data = ST::string::from_utf8((const char*)buf, dataSize);
+    PyObject* result = PyString_FromSTString(data);
```

Comment:
You could use something like `PyString_FromStringAndSize` to prevent copying into a temporary `ST::string`.

Reply:
Nifty!